sdk/app_cpu1/common/drv/timing_manager.h

```diff
@@ -18,14 +18,7 @@
 #define TIMING_MANAGER_BASE_ADDR (XPAR_AMDC_TIMING_MANAGER_0_S00_AXI_BASEADDR)
 #define UPPER_16_SHIFT           (16)
 
-typedef enum {
-	EDDY_0 = 0,
-	EDDY_1 = 1,
-	EDDY_2 = 2,
-	EDDY_3 = 3,
-	ENCODER = 4,
-	ADC = 5
-} sensor_t;
```

Comment:
FYI--if you like the vertical spacing/format here, I think you can keep it even in the auto-format results. Most auto formatter tools (I think including clang-format) support a trailing `,` comma which indicates the list should be line-by-line. So, simply add a comma to the ADC line: `ADC = 5,` and it should autoformat into what you originally had in the previous commit.

You don't need to change this if you don't want to, but thought I'd tell you so you're aware.